src/main/java/com/revealprecision/revealserver/RevealServerApplication.java

```diff
@@ -15,6 +16,7 @@
 @EnableJpaAuditing
 @Slf4j
 @EnableConfigurationProperties
+@ServletComponentScan
```

Comment:
Is this being used with the current checkin? what are we scanning?

Reply:
This leaked in....it was meant to be with the fhir facade code....i had to redo this branch....

Il remove and ship this with the FHIR branch.....